File: app/src/main/java/com/nytimes/android/external/playbillingtester/model/Repository.java
package com.nytimes.android.external.playbillingtester.model;

import org.immutables.gson.Gson;
import org.immutables.value.Value;

import java.util.Date;

@Value.Immutable
@Gson.TypeAdapters(fieldNamingStrategy = true)
public interface Repository {
    String name();
    String fullName();
    String description();
    Date pushedAt();
    int stargazersCount();
    int forksCount();
    String language();
    String htmlUrl();
}

File: app/src/main/java/com/nytimes/android/external/register/model/Repository.kt
package com.nytimes.android.external.register.model

import java.util.*

data class Repository(
    val name: String,
    val fullName: String,
    val description: String,
    val pushedAt: Date,
    val stargazersCount: Int,
    val forksCount: Int,
    val language: String,
    val htmlUrl: String
)
